imblearn/keras/generator.py

```diff
@@ -0,0 +1,226 @@
+"""Implement generators for ``keras`` which will balance the data."""
+from __future__ import division
+
+try:
+    import keras
+except ImportError:
+    # Skip the tests for the examples
+    import pytest
+    keras = pytest.importorskip('keras')
```

Comment:
This line in my box generates the following error:
```python
Skipped: could not import 'keras'
```

Reply:
I wanted to use it to skip the test of the docstring but probably there is something better to do then.